packages/spectacle/src/components/deck/deck.tsx

```diff
@@ -400,21 +400,16 @@ export const DeckInternal = forwardRef<DeckRef, DeckInternalProps>(
     //
     // Yes, this is slightly awkward, but IMO adding an additional `<div>` element
     // would be even more awkward.
-    let useFallbackBackdropStyle = true;
-    const backdropStyle = themeProvidedBackdropStyle;
-    let BackdropComponent = 'div' as ElementType;
-    if (userProvidedBackdropStyle) {
-      Object.assign(backdropStyle, userProvidedBackdropStyle);
```

Comment:
Probably never happened to a real user in a way that would surface as a bug, but this `Object.assign` call does an in-place modification of the object the user passes into `theme.backdropStyle`.